src/signing/kms-signer.ts

```diff
@@ -1,7 +1,32 @@
+import {KMSClient, SignCommand, SignCommandInput} from '@aws-sdk/client-kms';
 import {Signer} from '.';
 
 export class KmsSigner extends Signer<string> {
-  sign(): string {
-    throw new Error('Method not implemented.');
+  keyId: string;
+  kmsClient: KMSClient;
+
+  constructor(keyId: string) {
+    super();
+    this.keyId = keyId;
+    this.kmsClient = new KMSClient();
+    console.log('create client with key id', keyId);
```

Comment:
Might not want to write this to the log on CI 💭

Reply:
yup good spot will remove